styles.css

```diff
@@ -111,3 +111,105 @@
 .main__button:hover {
   opacity: .5;
 }
+
+@media (max-width: 1560px) {
+  .wrapper {
+    width: 69%;
+}
+
+.menu__link {
+  font: 500 14px/16px 'Roboto', sans-serif, arial, helvetica;
+}
+
+  .main {
+  padding: 0 58% 100px 0;
+  background: top right no-repeat url(img/mainimg.svg);
+}
+
+.main__title {
+  font: bold 50px/66px 'PT Serif', serif, sans-serif, arial, helvetica;
+}
+
+.main__title_highlight {
+  background: #ecf3ff;
+}
+
+.main__text {
+  font: 500 18px/21px 'Roboto', sans-serif, arial, helvetica;
+  width: auto;
+  height: auto;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html-css-2-0/pullRequest?prid=6306727)